tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/CI/TestingFrameworkImpactedTests.cs

```diff
@@ -0,0 +1,278 @@
+                SetEnvironmentVariable(ConfigurationKeys.CIVisibility.ImpactedTestsDetectionEnabled, enabled.Value ? "True" : "False");
+            }
+
+            static string GetEventJsonFile()
+            {
+                string content = $$"""
+                {
+                  "pull_request": {
+                    "head": {
+                      "sha": "{{GitHubSha}}"
+                    },
+                    "base": {
+                      "sha": "{{GitHubBaseSha}}"
+                    }
+                  }
+                }
+                """;
+                var tmpFileName = Path.Combine(Path.GetTempPath(), Guid.NewGuid().ToString() + "_event.json");
+                File.WriteAllText(tmpFileName, content);
+                return tmpFileName;
+            }
+        }
+
+        private MockTracerAgent GetAgent(List<MockCIVisibilityTest> receivedTests, Action<MockTracerAgent.EvpProxyPayload, List<MockCIVisibilityTest>> processRequest = null)
+        {
+            var agent = EnvironmentHelper.GetMockAgent();
+            agent.EventPlatformProxyPayloadReceived += (sender, e) =>
+            {
+                if (processRequest != null)
+                {
+                    processRequest(e.Value, receivedTests);
+                    return;
+                }
+
+                ProcessAgentRequest(e.Value, receivedTests);
+            };
+
+            return agent;
+        }
+
+        private void InitGit()
+        {
+            // Check git availability
+            var output = RunGitCommandAsync("branch --show-current");
+            if (output.ExitCode < 0)
+            {
+                // Try to fix the git path
+                RunGitCommandAsync("config --global --add safe.directory '*'");
+                output = RunGitCommandAsync("branch --show-current");
+            }
+
+            if (output.ExitCode == 0)
+            {
+                gitAvailable = true;
+                Output.WriteLine($"Git NOT available. ExitCode: {output.ExitCode} Error: {output.Error}");
+            }
+        }
+
+        private ProcessHelpers.CommandOutput RunGitCommandAsync(string arguments)
+        {
+            try
+            {
+                var sw = System.Diagnostics.Stopwatch.StartNew();
+                var gitOutput = AsyncUtil.RunSync<ProcessHelpers.CommandOutput>(() => ProcessHelpers.RunCommandAsync(
+                                    new ProcessHelpers.Command(
+                                        "git",
+                                        arguments,
+                                        EnvironmentTools.GetSolutionDirectory(),
+                                        outputEncoding: Encoding.Default,
+                                        errorEncoding: Encoding.Default,
+                                        inputEncoding: Encoding.Default,
+                                        useWhereIsIfFileNotFound: true),
+                                    null));
+
```

Comment:
There's no async version anymore, so you can avoid the AsyncUtil... Just use the Sync version